Day7/arrayReverse.js

```diff
@@ -0,0 +1,12 @@
+function reverse(arr) {
+  let inReverse = arr.reverse();
+  let space = "";
+
```

Comment:
:bulb:  You can kindly consider using a descriptive variable names like `reversedArrayString` which is easier to understand for someone looking at your code. And make sure to use `cammel case` notation if you intend to use more than one word for a variable name